ksql-engine/src/test/java/io/confluent/ksql/integration/EndToEndIntegrationTest.java

```diff
@@ -83,6 +86,9 @@
   @Before
   public void before() throws Exception {
     testHarness = new IntegrationTestHarness();
+    IntegrationTestHarness.consumedCount.set(0);
```

Comment:
Be nice if we can avoid exposing these statics. I realize why they are there, i.e. to get around the fact the interceptors are instantiated via reflection at some latter time, but maybe we can hide this within the `IntegrationTestHarness` better? e.g. `IntegrationTestHarness` can have a static `THIS` field that points to the last instantiated instance, and the interceptors could use this. Then the produce and consume counts can be a field on the `IntegrationTestHarness`.  It's not much better, (still uses static / still would cause problem if multiple instances are in play), but at least the use of it in tests is cleaner.